.github/workflows/genai_package.yml

```diff
@@ -38,7 +38,6 @@ jobs:
         if: ${{ 'Release' != matrix.build-type }}
       - run: source ./ov/setupvars.sh && timeout 25s ${{ github.workspace }}/s\ pace/samples_bin/greedy_causal_lm ./TinyLlama-1.1B-Chat-v1.0/ ""
       - run: source ./ov/setupvars.sh && timeout 25s ./ov/samples/python/multinomial_causal_lm/multinomial_causal_lm.py ./TinyLlama-1.1B-Chat-v1.0/ 0
-        if: ${{ 'Release' == matrix.build-type }} # Python bindings can be built in Release only
```

Comment:
does Python bindings have **d** suffix at the end? 
In yes, I wonder how it works because filename is used to import module in Python code.

Reply:
`pybind11_add_module()` known to tweak settings to generate a better binary. It probably ensures that debug bindings can be imported.

Reply:
- looks like debug samples hangs on Windows
- here you build w/o OV Dev package and don't have **d** suffix